### PR TITLE
Update READMEs and ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by taneliang**
1. Remove dead badges in main README
2. Move all badges to main README
3. Describe monorepo in main README
4. Remove contact info in www/README since that info is already in main README
5. Add links to deploys in www/README
6. s/video recording/link to video recording/ in ISSUE_TEMPLATE
6. Rephrase some things in all 3 files

### CODE REVIEW

File: www/README.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 # NUSMods R [![Coverage Status](https://coveralls.io/repos/github/nusmodifications/nusmods/badge.svg?branch=master)](https://coveralls.io/github/nusmodifications/nusmods?branch=master) [![Build Status](https://travis-ci.org/nusmodifications/nusmods.svg?branch=master)](https://travis-ci.org/nusmodifications/nusmods)
```
Comment (yangshun) [inline www/README.md]: I think we can remove the badges in the non-root READMEs.

File: www/README.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-- Facebook: https://www.facebook.com/nusmods
-- Messenger: https://www.m.me/nusmods
-- Twitter: https://twitter.com/nusmods
-- Email: nusmods@googlegroups.com
+- Production deployment: https://nusmods.com/
+- Latest build: https://latest.nusmods.com/
+- Issues: https://github.com/nusmodifications/nusmods/issues?q=is%3Aissue+is%3Aopen
 
-NUSMods R is built using [React][react], [Redux][redux], [Bootstrap][bootstrap] and is designed to be **fast, modern and responsive**.
+For the previous version NUSMods V2's documentation on installation, refer [here](../provisioning/README.md).
```
Comment (yangshun) [inline www/README.md]: > For the previous version NUSMods V2's documentation

This sentence reads oddly...

File: www/README.md [hunk 3]
```diff
@@ -24,7 +22,7 @@ Mobile browsers:
 
 ## Contributing
 
-Don't know where to start? One of our contributors, wrote [this fantastic beginner guide][zames-guide], it is highly recommended that you have a read.
+Don't know where to start? We highly recommend reading [this fantastic beginner's guide][zames-guide] written by one of our contributors. Alternatively, have a poke at our [open issues](https://github.com/nusmodifications/nusmods/issues?q=is%3Aissue+is%3Aopen).
```
Comment (yangshun) [inline www/README.md]: Do we wanna mention issues with `good-for-newcomers` tag here? Or maybe we don't have enough volume.

Comment (taneliang) [reply www/README.md]: Yeah that tag is so rarely used. I don't think we should link to that at the moment

**Pull request merged**